src/Panel/panel.scss

```diff
@@ -1,4 +1,13 @@
-@import '../utils/theme/variables.scss';
+$border-color: #e6e6e6 !default;
+$soft-loader-color: #45a7b9 !default;
+$panel-header-background-color: #2c3a40 !default;
```

Comment:
please remove the prefix `panel-` from the variables as it's implicit (we're not adding a prefix in the others components)